lib/awful/tag.lua

```diff
@@ -1370,11 +1366,6 @@ object.properties(capi.tag, {
     setter_fallback = tag.setproperty,
 })
 
--- fix a load loop
-client = require("awful.client")
-capi.client.connect_signal("manage", function(c) client.object.to_selected_tags(c) end)
```

Comment:
It cannot stay. Delayed timer make the urgent and focus tests fail, so it doesn't work
also, it cause the "tag" property to fail due to a race condition, new_tag break too. See the comments in awful.rules and PR description.

It would rather make awful.rules mandatory. It doesn't conflict with neither Shifty nor Tyrannical, so what's the point of having this optional?